python/xgrammar/matcher.py

```diff
@@ -400,3 +400,122 @@ def _debug_print_internal_state(self) -> str:
             The internal state of the matcher.
         """
         return self._handle._debug_print_internal_state()
+
+    @staticmethod
+    def batched_fill_next_token_bitmask(
+        matchers: List["GrammarMatcher"],
+        bitmask: ArrayLike,
+        index: int = 0,
+        max_threads: int = 16,
+        debug_print: bool = False,
+    ) -> List[bool]:
```

Comment:
```suggestion
    @staticmethod
    def batch_fill_next_token_bitmask(
        matchers: List["GrammarMatcher"],
        bitmask: ArrayLike,
        indices: Optional[List[int]] = None,
        max_threads: Union[int, Literal["auto"]] = "auto",
        debug_print: bool = False,
    ) -> None:
```
In docstring: if auto: half of the logical cores